Documentation/t5-model/README.md

```diff
@@ -0,0 +1,30 @@
+#Training Time Estimation
+
+## Training infos T5-Small:
+- GPU: NVIDIA GeForce GTX 960: 
+- Parameters: 60 million 
+- Epoch: 4 
+- Dataset: 100
+
+![Monitor](monitoring.png)
+
+
+## Training infos T5-Large:
+- 4 GPUs with 48 GB VRAM each
+- Parameters: 220 million
+- Epoch: 4
+- Dataset: 1000
+
+### Assumptions for Training AI to create Tickets:
+- Dataset Scaling: 100 to 10,000  =>  100 times larger.
+- Time per Epoch:  1 minutes per epoch => approximately 1.7 hours per epoch.
```

Comment:
Shouldn't it be 1 min per batch in one epoch?

Reply:
![monitoring](https://github.com/amosproj/amos2023ws01-ticket-chat-ai/assets/49998663/56459663-a54d-469d-a64d-afe10c26ba26)
as one can see on the right in this picture. It is 53,38 s per epoch. On left side in the pip Epoch 1/4 means First Epoch out of four.

Reply:
I also pushed the updated code for training. 
![grafik](https://github.com/amosproj/amos2023ws01-ticket-chat-ai/assets/49998663/7112d05c-4d72-4d2c-865c-f432a10c459d)
In this pic one can see that each iteration in the for loop is an epoch and the epoch time is the curent time minus start-time of the epoch